packages/react/src/components/FnxFloatingButton/pages/permits/GeneratePermitPage/components/ExpirySection.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import clsx from 'clsx';
+import { cn } from '../../../../../../utils/cn.js';
```

Comment:
Two proposals from my side:
1. let's use alias to have shorter paths:
2. let's get rid of `.js` postfix, it's confusing 
```suggestion
import { cn } from '@/utils/cn';
```